help/submit.md

```diff
@@ -145,8 +145,10 @@ Availability of submissions and announcement order
 Submissions to arXiv are typically made public Sunday through
 Thursday, with no announcements Friday or Saturday. Check the [current
 time](https://arxiv.org/localtime) at the arXiv main site, with the
-amount of time remaining before the deadline.
+amount of time remaining before the deadline. 
 
+The final [arXiv identifier](arxiv_identifier) cannot be assigned until the paper is announced (so please do not ask for it 
+ahead of announcement). It may be the case that a submission appears in a different identifier month due to various factors. **Note:** The arXiv identifier cannot be back-dated, and identifiers are ordered based on the final timestamp of submitter action (i.e. when "submit" is clicked). 
 

```

Comment:
Maybe also add a note about unsubmitting here? Something like "Any changes after clicking "submit, for example adding a cross-link, will require unsubmitting first, which will remove the submission from the submit order. A new timestamp will be assigned when "submit" is clicked again."

Reply:
part of the problem is that we're talking about two different systems, since it's possible in classic to make changes without unsubmitting first. I'm not sure if we want to explicitly state that it's required (yet).

Reply:
I see. If its not 100% accurate we shouldnt add that statement.